nipoppy/workflow/dicom_org/check_dicom_status.py

```diff
@@ -52,7 +53,7 @@ def run(global_config_file, regenerate=False, empty=False):
     # load manifest
     fpath_manifest = dpath_dataset / FPATH_MANIFEST_RELATIVE
     df_manifest = load_manifest(fpath_manifest)
-    df_status = df_manifest.loc[~df_manifest[COL_BIDS_ID_MANIFEST].isna()].copy()
+    df_status = df_manifest.loc[~df_manifest[COL_SESSION_MANIFEST].isna()].copy()
```

Comment:
Does this imply that `session_id` is the only way to tell if subject has imaging data in the manifest? 
It might be good to have additional check for `datatype` as well? And raise warnings if either of them are empty?

Reply:
Yeah I was also thinking of adding the check for `datatype`. At one point we said we only wanted `session_id` to be defined if the subject had imaging data -- is that still the case?

For the warning, does that mean there should be a warning if a subject doesn't have imaging data? Because that can happen fairly often in PPMI, especially for non-baseline visits/sessions